packages/dd-trace/src/exporters/agent/writer.js

```diff
@@ -70,7 +75,20 @@ class Writer {
 
     log.debug(() => `Request to the agent: ${JSON.stringify(options)}`)
 
-    platform.request(Object.assign({ data }, options), (err, res) => {
+    platform.metrics().increment('datadog.tracer.node.exporter.agent.requests')
+
+    platform.request(Object.assign({ data }, options), (err, res, status) => {
+      if (status) {
+        platform.metrics().increment('datadog.tracer.node.exporter.agent.responses')
+        platform.metrics().increment('datadog.tracer.node.exporter.agent.responses', `status:${status}`)
```

Comment:
Why both? This will likely double count if you ever graph just `.responses`

Reply:
The second one is actually supposed to be `.by.status`.